menu.php

```diff
@@ -230,6 +230,8 @@
 			</b>
 
 		</div>
+		<script>
+</script>
```

Comment:
Is this just cruft? Looks like an empty script tag for no reason.

Reply:
whoops, yep cruft- will delete.